grails-app/views/index.gsp

```diff
@@ -1,26 +1,13 @@
 <html>
-<head>
+  <head>
     <meta name="layout" content="main">
+    <meta name="layout" content="minimal">
+    <meta name="layout" content="main_component">
```

Comment:
meta with same name (layout) makes any sense ? Keep only the meaningful one.

Reply:
Grails redirects to view/index, we need to keep the reference to the main_component from there.